pyproject.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
     "psycopg2-binary", # PostgreSQL
     "PyMySQL",         # MySQL & MariaDB
     "cryptography",    # MySQL & MariaDB dependency
-    "pyodbc",          # Microsoft SQL Server
+    "pyodbc>=4.0.35",  # SQL Server
```

Comment:
5.2.0 is the current version.  Is there a situation in which you think a pre 4.0.35 build could get installed?